pandas/tests/base/test_ops.py

```diff
@@ -557,66 +557,25 @@ def test_value_counts_datetime64(self, index_or_series):
         result2 = td2.value_counts()
         tm.assert_series_equal(result2, expected_s)
 
-    def test_factorize(self):
-        for orig in self.objs:
-            o = orig.copy()
-
-            if isinstance(o, Index) and o.is_boolean():
-                exp_arr = np.array([0, 1] + [0] * 8, dtype=np.intp)
-                exp_uniques = o
-                exp_uniques = Index([False, True])
-            else:
-                exp_arr = np.array(range(len(o)), dtype=np.intp)
-                exp_uniques = o
-            codes, uniques = o.factorize()
-
-            tm.assert_numpy_array_equal(codes, exp_arr)
-            if isinstance(o, Series):
-                tm.assert_index_equal(uniques, Index(orig), check_names=False)
-            else:
-                # factorize explicitly resets name
-                tm.assert_index_equal(uniques, exp_uniques, check_names=False)
-
-    def test_factorize_repeated(self):
-        for orig in self.objs:
-            o = orig.copy()
-
-            # don't test boolean
-            if isinstance(o, Index) and o.is_boolean():
-                continue
-
-            # sort by value, and create duplicates
-            if isinstance(o, Series):
-                o = o.sort_values()
-                n = o.iloc[5:].append(o)
-            else:
-                indexer = o.argsort()
-                o = o.take(indexer)
-                n = o[5:].append(o)
+    @pytest.mark.parametrize("sort", [True, False])
```

Comment:
i think we have a sort fixture

Reply:
We only have a sort fixture for the index setops in `tests/indexes/conftest.py`, but that one uses different values (`[None, False]`). There's nothing yet for [True, False] values. I could create a follow-up issue though